src/components/Grid/Grid.tsx

```diff
@@ -4,14 +4,21 @@ import Modal from '../Modal/Modal';
 import './Grid.scss';
 import { cellsMockData } from './cellsMockData';
 
-const Grid = () => {
+type GridProps = {
+  bullDoze: boolean;
+  setBullDoze: Function;
+  filterGarden: boolean;
+  setFilterGarden: Function;
+}
```

Comment:
could i ask for one more change please!!  Since this is getting pretty big now, could you try converting this to an interface and extend it as necessary in other places?  This would be great practice :)